sdk/monitor/monitor-opentelemetry-exporter/package.json

```diff
@@ -32,20 +29,15 @@
     "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
     "unit-test": "npm run unit-test:node && npm run unit-test:browser",
     "unit-test:browser": "echo skipped",
-    "unit-test:node": "dev-tool run test:node-tsx-ts -- --timeout 1200000 \"test/internal/**/*.test.ts\"",
-    "unit-test:node:debug": "dev-tool run test:node-tsx-ts -- --timeout 1200000 \"test/internal/**/*.test.ts\"",
+    "unit-test:node": "dev-tool run test:vitest",
```

Comment:
I want to add a mod to do this automatically, but do note that the timeouts need to be copied over manually. Mocha had one timeout but vitest has test-timeout and hook-timeout so either one might need to be updated

Reply:
Timeouts not necessary here at least and may just be legacy.